packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -46,7 +46,8 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
       value: _value ? 1.0 : 0.0
     );
     _position = new CurvedAnimation(
-      parent: _positionController
```

Comment:
this one maybe argues that it's ok to have it imply linear? We do set it later...